Branch-SDK/src/main/java/io/branch/referral/network/BranchRemoteInterfaceUrlConnection.java

```diff
@@ -57,9 +56,6 @@ private BranchResponse doRestfulGet(String url, int retryNumber) throws BranchRe
         PrefHelper prefHelper = PrefHelper.getInstance(branch.getApplicationContext());
         try {
             int timeout = prefHelper.getTimeout();
-            if (timeout <= 0) {
```

Comment:
`prefHelper.getTimeout` has a non-zero default value, so this would always fail/have no effect unless the client used `setNetworkTimeout` API and passed a negative value, which makes no sense.